### ISSUE TITLE
Question about weight decay setting

### ISSUE THREAD

**Issue by delaprada** [microsoft/MeshTransformer#87]
Hi, thank you for providing such good work.

I have question about the weight_decay setting in `run_metro_handmesh.py`. The weight_decay is strictly set to 0 for Adam optimizer. But according to the [Freihand training reproduced log](https://datarelease.blob.core.windows.net/metro/models/freihand_reproduce_log.txt) provided in `EXP.md`, the weight_decay is 0.05.

May I ask which setting is more appropriate?
Thank you!
